src/ios/FirebaseDynamicLinksPlugin.h

```diff
@@ -2,6 +2,7 @@
 #import "AppDelegate.h"
 
 @import Firebase;
```

Comment:
Thanks, it's planned to get rid of `@import Firebase` after upgrading to v9 of iOS SDK https://firebase.google.com/support/release-notes/ios#version_900_-_may_3_2022